torch_glow/src/GlowFuser.cpp

```diff
@@ -185,26 +185,33 @@ c10::optional<torch::jit::Node *> tryMerge(torch::jit::Node *consumer,
 }
 #undef REQ
 
-torch::jit::graph_node_list::iterator
+std::pair<torch::jit::graph_node_list::iterator, bool>
 getNewNode(torch::jit::Node *node, torch::jit::AliasDb &aliasDb,
            torch::jit::Block *block, isSupportFunc fn, at::Symbol kind) {
   auto node_inputs = sortReverseTopological(node->inputs(), block);
   for (auto input : node_inputs) {
     if (auto group = tryMerge(node, input->node(), aliasDb, fn, kind)) {
-      return group.value()->reverseIterator();
+      return {group.value()->reverseIterator(), true};
     }
   }
-  return ++node->reverseIterator();
+  return {++node->reverseIterator(), false};
 }
 
 void GlowCustomFuse(std::shared_ptr<torch::jit::Graph> graph, isSupportFunc fn,
                     at::Symbol kind) {
   torch::jit::AliasDb aliasDb(graph);
   auto block = graph->block();
 
-  for (auto it = block->nodes().rbegin(); it != block->nodes().rend();) {
-    it = getNewNode(*it, aliasDb, block, fn, kind);
-  }
+  bool is_changed;
+  do {
```

Comment:
why not just use a while loop like in https://github.com/pytorch/tvm/pull/88/files?

Reply:
o it is same thing...